.babelrc

```diff
@@ -1,4 +1,11 @@
 {
-  "presets": ["es2015", "react", "stage-0"],
-  "plugins": ["transform-flow-strip-types", "transform-decorators-legacy", "react-hot-loader/babel"]
+  "presets": [
+    "es2015",
+    "react",
+    "stage-0"
+  ],
+  "plugins": [
+    "transform-flow-strip-types",
+    "transform-decorators-legacy"
+  ]
```

Comment:
This is all just prettifying stuff and most importantly removing `react-hot-loader/babel`